### PR TITLE
Fix add new open models

### PR DESCRIPTION
**Pull request by nightingal3**
## Description

FIx of this PR https://github.com/nightingal3/llm-pretraining-behaviours/pull/126 which had too many files changed. The problem actually seemed to be that log files (e.g. logits from evaluating the models, logs from xgboost) were committed, causing a large number of changed files. 

## References

- Link https://github.com/nightingal3/llm-pretraining-behaviours/pull/126

### CODE REVIEW

**Review by neubig — APPROVED** [review]
Looks good to me for now, thanks!
